core/controllers/tasks.py

```diff
@@ -131,6 +132,25 @@ def post(self):
 class InstantFeedbackMessageEmailHandler(base.BaseHandler):
     """Handles task of sending feedback message emails instantly."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'user_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'reference_dict': {
+                'schema': {
+                    'type': 'object_dict',
+                    'object_class': feedback_domain.FeedbackMessageReference
```

Comment:
Hi @MattiasErlingson You need to write a from_dict method in [FeedbackMessageReference](https://github.com/oppia/oppia/blob/develop/core/domain/feedback_domain.py#L329) class.

Take a look at this method as guidance: [from_dict for Exploration class](https://github.com/oppia/oppia/blob/ef23203792be046abc6fbab569b645a36eb24706/core/domain/exp_domain.py#L1437).

Let me know if you need any help regarding this.
Thank you

Reply:
Thank you for the help! I'll look at it immediately.

Reply:
A question arose @Nik-09, does the FeedbackMessageReference need a validate method as well? If so, are there any examples for those as a reference as well?

Reply:
Responded in the below comment